### DISCUSSION TITLE
Newsletter plattforms

### DISCUSSION THREAD

**Discussion by treeandcoffee** [GetPublii/Publii#1837] [Help] [answered]
Hi 
Which newsletter plattforms do you use with Publii and how you integrate them into publii?

**Comment by carstenfeller** [comment]
Please see https://github.com/GetPublii/Publii/discussions/1842#discussioncomment-11447323 to use the "Custon HTML" Plugin.

If you use Mailchimp, they will give you a code for the sign-up-form: https://mailchimp.com/help/add-a-signup-form-to-your-website/

You maybe want to embedd this code in the footer or after every post via plugin or within the content via editor.

I can recommend CleverReach for EU/ DSGVO sensitive Websites or Mailchimp.

**Reply by treeandcoffee** [reply]
Thank you very much for your response.

**Comment by candidexmedia** [comment]
# What I Do
I self-host [Listmonk](https://listmonk.app/), an open source newsletter manager, using Railway.app ([tutorial](https://x.com/candidexmedia/status/1623032874172616708)), and embed the signup form in Publii.

To do so, I embed the plain, unstyled form ([example demo](https://demo.listmonk.app/admin/lists/forms)) in my footer using [theme overrides](https://getpublii.com/dev/theme-overrides/):
- I add the form embed code to my `footer.hbs`
- I style the form by editing my `main.css` file. Otherwise, it inherits the existing form styling

I use Listmonk because it's FOSS and the most affordable option (you can use it with whatever SMTP service you like), but if I didn't use that, I'd look into buttondown, EmailOctopus, or beehiiv.

# Ways of Embedding Newsletter Signup Forms in Publii

There are, of course, easier ways of going about this. Nearly all newsletter platforms offer some kind of signup form embed solution:

## iframe

Some will offer an iframe that can be plugged in anywhere that accepts HTML (ex: your post/page editors, [Custom HTML tool](https://getpublii.com/docs/tools.html#customhtml))

## Form Action

Some, like Listmonk, will offer a `<form>` with an `action=` attribute that sends the submitted data to the newsletter service. These can also be plugged in anywhere that accepts HTML (ex: your post/page editors, [Custom HTML](https://getpublii.com/docs/tools.html#customhtml)), and styling can be done easily via [Custom CSS](https://getpublii.com/docs/tools.html#customcss)

## JavaScript

Some will offer a `<script>` which can dynamically generate the form (think timed pop-up). Those will either go in the `<head>` or at the end of the `<body>`, and can be added via [Custom HTML](https://getpublii.com/docs/tools.html#customhtml)


Last note: these tips work all kinds of embedded third-party content, not just newsletters.

**Comment by treeandcoffee** [comment]
Hi 
Many thanks for your answer and explanation. Your solution with a self-hosted FOSS newsletter tool and subsequent connection to a sender platform is a really good idea. 
At first I thought about reactivating my FOSS [Sendy](https://sendy.co/) instance, but I would have to configure it instead of AWS SES to my choosen EC/EU-based [Emailit](https://emailit.com/) first because I only have an API connection there. I could probably achieve this via Listmonk. Of all the platforms you mentioned, I was only aware of EmailOctopus. Thank you very much for your extensive input.

**Reply by candidexmedia** [reply, marked as answer]
No problem! I forgot to include the link to a tutorial I made a while back on how to deploy Listmonk, so [here's the link](https://x.com/candidexmedia/status/1623032874172616708).
